### DISCUSSION TITLE
Making a sound pack for Dreame L10 Pro

### DISCUSSION THREAD

**Discussion by bunglesss** [rand256/valetudo#520] [General]
Does anyone know how to do it?

I tried sftp all the English sounds off it, then swapping track 7 which I think is "start cleaning", with the following commands. The upload failed at the end of it all.

scp -rO root@192.168.x.x:/audio/EN Downloads/
ffmpeg -i "7.wav" -filter:a loudnorm=I=-14:LRA=1:dual_mono=true:tp=-1 "7_n.wav" && oggenc "7_n.wav" --output "7.ogg" --bitrate 100 --resample 16000
tar czf ../voicepapa.tar.gz *.ogg
md5sum ../voicepapa.tar.gz

I want the hoover to say "Mr Zurkon is here to kill" when it starts, from Ratchet and Clank.

**Comment by pidator** [comment]
You're asking in the wrong repo, this fork of valetudo doesn't support Dreame robots.
